secboot/secboot_tpm.go

```diff
@@ -219,7 +231,7 @@ func UnlockVolumeIfEncrypted(disk disks.Disk, name string, encryptionKeyDir stri
 				// volumes to unlock we should lock access to the sealed keys only after
 				// the last encrypted volume is unlocked, in which case lockKeysOnFinish
 				// should be set to true.
-				lockErr = sbLockAccessToSealedKeys(tpm)
+				lockErr = sbBlockPCRProtectionPolicies(tpm, []int{tpmPCR})
```

Comment:
we should discuss with Chris whether is best to scramble just one PCR or all the relevant ones

Reply:
We should only touch the one that we've currently reserved for the kernel EFI image, so the current code looks correct. Touching others will break the ability to perform any kind of attestation using the TPM because it will make the log inconsistent.

Reply:
thanks for the input.  I suppose we'll want some comments around that and a better name, like initramfsPCR or something like that